packages/js/app/package.json

```diff
@@ -15,8 +14,28 @@
         "test": "jest"
     },
     "dependencies": {
+        "@fortawesome/fontawesome-svg-core": "^6.1.1",
+        "@fortawesome/free-solid-svg-icons": "^6.1.1",
+        "@fortawesome/react-native-fontawesome": "^0.3.0",
+        "@react-native-community/masked-view": "*",
+        "@react-navigation/bottom-tabs": "6.3.1",
+        "@react-navigation/drawer": "6.4.1",
+        "@react-navigation/elements": "1.3.3",
+        "@react-navigation/material-bottom-tabs": "6.2.1",
+        "@react-navigation/material-top-tabs": "6.2.1",
+        "@react-navigation/native": "6.0.10",
+        "@react-navigation/native-stack": "6.6.1",
+        "@react-navigation/stack": "6.2.1",
         "react": "17.0.2",
-        "react-native": "0.68.2"
+        "react-native": "0.68.2",
+        "react-native-gesture-handler": "~2.1.0",
+        "react-native-pager-view": "5.4.9",
+        "react-native-paper": "^4.7.2",
+        "react-native-reanimated": "~2.3.1",
+        "react-native-safe-area-context": "3.3.2",
+        "react-native-screens": "~3.10.1",
+        "react-native-svg": "^12.3.0",
+        "react-native-tab-view": "^3.0.0"
```

Comment:
I don't think all these dependencies are needed. Remove ones not needed.

Reply:
Just removed the unnecessary dependencies